railties/lib/rails/generators/rails/app/templates/Gemfile.tt

```diff
@@ -32,6 +32,10 @@ gem "bootsnap", require: false
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin Ajax possible
 # gem "rack-cors"
 <%- end -%>
+
+# Installing irb as a gem helps keeping it up-to-date
```

Comment:
My gut feel is the same as yahonda's here. But if we do go ahead with this, I think we should think of a different comment here. This explains to other Rails developers why irb was added, but I don't think it says much to Rails users. Compare to most of the other comments in the Gemfile template, which describe why the gem is there and what you can use it for.


```suggestion
# irb powers the Rails console [https://guides.rubyonrails.org/command_line.html#bin-rails-console]
```

Reply:
That's a good call, thank you 👍